profiles/common/modules/features/solr_config/solr_config.module

```diff
@@ -6,26 +6,34 @@
 
 include_once 'solr_config.features.inc';
 
+/**
+ * Implements hook_apachesolr_exclude().
+ */
 function solr_config_apachesolr_exclude($entity_id, $entity_type, $row, $env_id) {
   if ($entity_type == 'file') {
     $node = node_load($row->parent_entity_id);
 
-    if (isset($node->workbench_moderation)) { // node moderate with workbench moderation
+    if (isset($node->workbench_moderation)) {
+      // Node moderate with workbench moderation.
       $workflow = $node->workbench_moderation;
-      if (isset($workflow['published'])) { // node parent is published
+      if (isset($workflow['published'])) {
+        // Node parent is published.
```

Comment:
These two comments actually refer to the line above where they are placed. It's better to move them one line higher :)